docs/tutorial.rst

```diff
@@ -167,49 +170,60 @@ Create ``config/polls.yaml`` file with database schemas
       minsize: 1
       maxsize: 5
 
-    host: 127.0.0.1
-    port: 8080
-
 Install ``pyyaml``::
 
     $ pip install pyyaml
 
-And now load config into the application::
+Let's also create separate settings file. It helps to leave ``main.py`` clean and short::
 
-    # main.py
+    # settings.py
     import pathlib
-
-    from aiohttp import web
     import yaml
-    from routes import setup_routes
-
 
     BASE_DIR = pathlib.Path(__file__).parent.parent
     config_path = BASE_DIR / 'config' / 'polls.yaml'
-    with open(config_path) as f:
-        config = yaml.load(f)
+
+    def get_config(path):
+        with open(path) as f:
+            config = yaml.load(f)
+        return config
+
+    config = get_config(config_path)
```

Comment:
This approach leads to config singleton. Is it possible to avoid the singleton usage?
I makes isolation better, helpful for testing etc

Reply:
I considered this state of a file as a first stepping stone.
Since we move from empty folder and no database to the current repo state.

Then we can introduce changes and make arguments on why we do them.

Reply:
Aaah. Ok